src/style.css

```diff
@@ -0,0 +1,43 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  /* width: fit-content; */
+  align-items: center;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
```

Comment:
please do not set the `margin-right`  for last child. here you can go like this:
```css
.stars__star:not(:last-child) {
  margin-right: 4px;
}
```

Reply:
If I remove the margin-right for last child, six autometed tests are failed. I cann't do it.